integrations/webchat/src/definitions/channels.ts

```diff
@@ -0,0 +1,52 @@
+import { IntegrationDefinitionProps, messages } from '@botpress/sdk'
+import { z } from 'zod'
+
+const cardSchema = z.object({
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏ You should override the property instead of redeclaring it all together:
```
const cardSchema = messages.defaults.card.schema.extend({
  subtitle: z.string().optional(),
})
```